indico/modules/events/abstracts/controllers/management.py

```diff
@@ -113,24 +113,41 @@ def _process(self):
 class RHManageAbstractReviewing(RHManageAbstractsBase):
     """Configure abstract reviewing."""
 
+    @property
+    def ratings(self):
+        return (AbstractReviewRating.query
+                .join(AbstractReviewRating.review)
+                .join(AbstractReview.abstract)
+                .join(AbstractReviewRating.question)
+                .filter(~Abstract.is_deleted, Abstract.event == self.event,
+                        AbstractReviewQuestion.field_type == 'rating'))
+
+    def _scale_ratings(self, scale_min, scale_max):
+        prev_min = abstracts_reviewing_settings.get(self.event, 'scale_lower')
+        prev_max = abstracts_reviewing_settings.get(self.event, 'scale_upper')
+        if scale_min == prev_min and scale_max == prev_max:
+            return
+        assert scale_max > scale_min
+
+        abstracts_reviewing_settings.set(self.event, 'scale_lower', scale_min)
+        abstracts_reviewing_settings.set(self.event, 'scale_upper', scale_max)
```

Comment:
```suggestion
        abstracts_reviewing_settings.set_multi(self.event, {
            'scale_lower': scale_min,
            'scale_upper': scale_max,
        })
```

Reply:
Actually, I think this can completely go away since the `set_multi` right after the call to `self._scale_ratings()` should already save this...